src/main/java/bacnet/LocalBacnetPoint.java

```diff
@@ -219,9 +202,15 @@ private void setupPresentValueProperty(PropertyIdentifier pid) {
 			presentValueProperty = new LocalCharacterStringProperty(objectId, pid, this, node, presentValueNode);
 			break;
 		}
+		case BOOLEAN: {
+			presentValueProperty = new LocalBooleanProperty(objectId, pid, this, node, presentValueNode);
+			break;
+		}
 		}
```

Comment:
that switch/if indentation looks weird. Don't know if your IDE formats it like this or what.